unittests/photonics/PhotonicsTester.cpp

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+#include <gtest/gtest.h>
+
+#include "cudaq.h"
+#include "cudaq/qis/managers/photonics/photonics_qis.h"
```

Comment:
We should add a header in `runtime/cudaq/` called `photonics.h` that includes this "more complicated" photonics_qis.h path. The user would then only have to `#include "cudaq/photonics.h"`, much simpler that way.

Reply:
Added the header runtime/cudaq/photonics.h